fastapi-app/models/ingredient.py

```diff
@@ -0,0 +1,24 @@
+import uuid
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy import Column, String, DateTime
+from sqlalchemy.orm import relationship
+
+from db.base_class import Base, UtcNow
+
+
+class Ingredient(Base):
+    id = Column(
```

Comment:
If a line is too long and you have to split parameters into another row, put each parameter in a new row for better visibility. This goes for other changes in this PR as well.